src/libs/Navigation/AppNavigator/getRootNavigatorScreenOptions.js

```diff
@@ -12,59 +12,63 @@ const commonScreenOptions = {
     animationTypeForReplace: 'push',
 };
 
-export default (isSmallScreenWidth) => ({
-    rightModalNavigator: {
-        ...commonScreenOptions,
-        cardStyleInterpolator: (props) => modalCardStyleInterpolator(isSmallScreenWidth, false, props),
-        presentation: 'transparentModal',
+export default (isSmallScreenWidth, isHomeOdSelected) => {
+    const offsetWidth = isHomeOdSelected ? variables.globalNavigationWidth : variables.sideBarWidth;
```

Comment:
In the future if we add more tabs that don't have a subnav, are we going to have to go through everywhere and change the `isHomeOdSelected` variable?

Reply:
Yeah, probably to something like `isScreenWithoutSubnavSelected`

Reply:
Well, actually I can change the name of this var in this pr